app/react/Viewer/actions/ocrActions.ts

```diff
@@ -0,0 +1,15 @@
+import api from 'app/utils/api';
+
+const postToOcr = async (filename: string) => {
+  await api.post(`files/${filename}/ocr`);
```

Comment:
It is correct that this function doesn't return anything? Are not doing anything with the response of the POST either to acknowledge it received or to catch any errors?

Reply:
The UX design (or the issue) specified that the button immediately changes to the "In queue" without waiting for the response. If I'm not wrong this is a UX pattern in which you predict the successful action. Now you say this, probably we still need to catch the error and "rollback" the UI, but since you still get an error notification and the UX design didn't specified it. We basically overlooked it.
If you think it's a need we can implement it, in other case we can iterate on it in the future.